graveler/ref/manager.go

```diff
@@ -41,7 +42,17 @@ func (m *Manager) GetRepository(ctx context.Context, repositoryID graveler.Repos
 	return repository.(*graveler.Repository), nil
 }
 
-func (m *Manager) CreateRepository(ctx context.Context, repositoryID graveler.RepositoryID, repository graveler.Repository, branch graveler.Branch) error {
+const (
+	firstCommitMsg = "lakeFS repository created!"
+)
+
+func (m *Manager) CreateRepository(ctx context.Context, repositoryID graveler.RepositoryID, repository graveler.Repository, token graveler.StagingToken) error {
+	firstCommit := graveler.Commit{
+		Message:      firstCommitMsg,
+		CreationDate: time.Now(),
```

Comment:
verity if you need to call UTC or the pg lib does that for you

Reply:
It is done by the `addCommit` method below.